app/database/models/server/channel.ts

```diff
@@ -66,6 +66,12 @@ export default class ChannelModel extends Model implements ChannelModelInterface
 
         /** A USER can create multiple CHANNEL (relationship is 1:N) */
         [USER]: {type: 'belongs_to', key: 'creator_id'},
+
+        /** A CHANNEL is associated with one CHANNEL_INFO**/
+        [CHANNEL_INFO]: {type: 'has_many', foreignKey: 'id'},
```

Comment:
Why `has_many`? It should only have one, right? The same for the settings.

Reply:
WatermelonDB does not support 1:1 relationship out of the box and this is how we have 'abstracted' the relationship.